static/index.md

```diff
@@ -33,6 +33,15 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
+The following rate limiting rules have been implemented on the API
```

Comment:
Few lines to explain why we are adding the rate limit would be good. something along the lines of "We limit the number of requests you can make to dp API's to protect our service against excessive use and denial of service attacks ."

Reply:
We also need to mention about the  429 response when we exceed the limit stated and about the Retry-header as mentioned in the trello ticket

Reply:
And can you mention both the domains as well